src/cascadia/TerminalApp/CascadiaSettings.h

```diff
@@ -52,6 +52,7 @@ class TerminalApp::CascadiaSettings final
     static const CascadiaSettings& GetCurrentAppSettings();
 
     winrt::Microsoft::Terminal::Settings::TerminalSettings MakeSettings(std::optional<GUID> profileGuid) const;
+    std::tuple<GUID, winrt::Microsoft::Terminal::Settings::TerminalSettings> BuildSettings(const winrt::TerminalApp::NewTerminalArgs& newTerminalArgs) const;

```

Comment:
NAK on `MakeSettings` and `BuildSettings`. This seems like a great case for an overload.